packages/x-license/src/utils/plan.ts

```diff
@@ -1,5 +1,5 @@
-export const LICENSE_SCOPES = ['pro', 'premium'] as const;
+export const PLAN_SCOPES = ['pro', 'premium'] as const;
 export const PLAN_VERSIONS = ['initial', 'Q3-2024'] as const;
 
-export type LicenseScope = (typeof LICENSE_SCOPES)[number];
+export type PlanScope = (typeof PLAN_SCOPES)[number];
```

Comment:
This is public API strictly speaking
We can either keep exporting an `LIcenseScope` next to `PlanScope`, or don't care because nobody would actually use this variable